packages/client-sdk-nodejs/src/internal/internal-auth-client.ts

```diff
@@ -208,10 +211,22 @@ export class InternalAuthClient implements IAuthClient {
       return new GenerateDisposableToken.Error(normalizeSdkError(err as Error));
     }
 
+    const tokenID = disposableTokenProps?.tokenID;
+    if (tokenID) {
```

Comment:
javascript has very liberal definitions for "truthiness", so it's always better to be very explicit about the condition you are checking for. In this case I think it would be `!== undefined`.